snippets/facets.liquid

```diff
@@ -484,15 +488,17 @@
     </facet-remove>
```

Comment:
I wonder if above we should also have a h2 wrap the content of `mobile-facets__open-wrapper` as on mobile I only see the `{amount} of products` announced as a heading but not `filter and sort`.
![](https://screenshot.click/21-10-98b9r-8wugd.png)

Reply:
Good point, otherwise this may be missed when navigating via headings.

```diff
<summary  class="mobile-facets__open-wrapper focus-offset" …>
-  <span class="mobile-facets__open">
+  <h2 class="mobile-facets__open">
    <!-- SVG -->
    <span class="mobile-facets__open-label button-label">Filter and sort</span>
-  </span>
+  </h2>
</summary>
```